metadata/add_plot_boundaries.Rmd

```diff
@@ -0,0 +1,142 @@
+---
+title: "Adding plot boundaries to BETYdb"
+author: "Kimberly Huynh"
+date: "5/20/2019"
+output: html_document
+---
+
+# Objective
```

Comment:
The ticket has the following text and I expected the steps to be described here. Are they elsewhere? "To describe the experiment associated with a site, you will need to provide the experiment name. Start date, end date, design, and description are optional. The experiment can be associated with treatments. Minimum required information to describe a treatment is the treatment name."

Reply:
Some of this general information can be found in the metadata upload tutorial. This tutorial only focuses on adding the plot boundaries.